content/kubernetes/re-clusters/upgrade-redis-cluster.md

```diff
@@ -78,62 +83,96 @@ customresourcedefinition.apiextensions.k8s.io/redisenterprisedatabases.app.redis
 deployment.apps/redis-enterprise-operator configured
 ```
 
-### Verify that the operator is running
+### Reapply other manual configurations
+
+When upgrading your cluster, you need to manually reapply the `ValidatingWebhookConfiguration` for the admission controller. See the [Enable the admission controller]({{<relref "/kubernetes/deployment/quick-start#enable-the-admission-controller">}}) step during deployment for more details.
```

Comment:
Should we mention that this step is optional, and is only required if the user has previously enabled the admission controller (or wishes to enable it now..).